src/Network/Network/VirtualRouterPeer/RemoveAzureRMVirtualRouterPeerCommand.cs

```diff
@@ -59,15 +59,15 @@ public partial class RemoveAzureRmVirtualRouterPeer : VirtualRouterBaseCmdlet
             ValueFromPipeline = true,
             HelpMessage = "The virtual router peer input object.")]
         [ValidateNotNullOrEmpty]
-        public PSVirtualRouterPeer InputObject { get; set; }
+        public PSBgpConnection InputObject { get; set; }
```

Comment:
this input has to be PSVirtualRouterPeer, please convert to PSBgpConnection underlying.